src/main/java/uk/gov/di/ipv/cri/common/library/service/JWTVerifier.java

```diff
@@ -32,6 +38,8 @@
 
 public class JWTVerifier {
 
+    private static final Logger LOGGER = LogManager.getLogger();
```

Comment:
This (now) unused field can be removed. Sonar has detected it as a code smell too